packages/vote/fpc/parameters.go

```diff
@@ -12,10 +12,14 @@ type Parameters struct {
 	SubsequentRoundsLowerBoundThreshold float64
 	// The upper bound liked percentage threshold used after the first round.
 	SubsequentRoundsUpperBoundThreshold float64
+	// The fixed liked percentage threshold used in last 'l2' rounds.
+	EndingRoundsFixedThreshold float64
 	// The amount of opinions to query on each round for a given vote context. Also called 'k'.
 	QuerySampleSize int
 	// The amount of rounds a vote context's opinion needs to stay the same to be considered final. Also called 'l'.
 	FinalizationThreshold int
+	// The amount of last rounds for the fixed threshold.
+	FixedEndingThreshold int
```

Comment:
can we call it TotalRoundsFinalization and TotalRoundsFixedThreshold instead of thresholds, TotalRoundsCoolingOffPeriod instead of CoolingOffPeriod?

Reply:
looks good, I renamed them